safe/definitions/minimum_needs.py

```diff
@@ -70,9 +68,9 @@ def _initializes_minimum_needs_fields():
             field_name = '{namespace}__{field_name}'.format(**format_args)
             # It's easier to debug if we have the real value from the
             # postprocessor. That's why I put a Double. ET.
-            field_type = QVariant.Double
+            field_type = qvariant_numbers
```

Comment:
I think this one is also related now : 
![screen shot 2017-02-27 at 15 49 57](https://cloud.githubusercontent.com/assets/1609292/23365831/d4882570-fd04-11e6-8b2f-811efa5f40a6.png)

I don't what we can do for these ones.